web/src/pages/dashboard.tsx

```diff
@@ -168,8 +169,21 @@ const DashboardPage = (props: Props): ReactElement => {
   };
 
   const renderQuickActions = (): ReactElement => {
-    const renderQuickActions = [];
-    const isHomeContractor = isHomeContractorIndustry(business?.profileData.industryId);
+    const quickActionsArray = [];
+    let renderStateContractingExternalLink;
+
+    if (business?.profileData.businessPersona === "OWNING") {
+      renderStateContractingExternalLink = !isConstructionSector(business?.profileData.sectorId);
+    }
+
+    if (business?.profileData.businessPersona !== "OWNING") {
+      renderStateContractingExternalLink =
+        !isConstructionSector(business?.profileData.sectorId) &&
+        !isHomeContractorIndustry(business?.profileData.industryId);
+    }
+    // const isHomeContractor = isHomeContractorIndustry(business?.profileData.industryId);
```

Comment:
[pebble] make sure to get rid of commented code before merging into main

Reply:
done